### ISSUE TITLE
Allow for setting the refferer policy security header

### ISSUE THREAD

**Issue by Crevil** [szepeviktor/fix-w3tc#435]
It would be great to be able to set the `Referrer-Policy` security header in the Security Headers section of the Browser cache settings.

Link to the w3 specification: [https://www.w3.org/TR/referrer-policy/](https://www.w3.org/TR/referrer-policy/)

The possible values are:
```
enum ReferrerPolicy {
  "",
  "no-referrer",
  "no-referrer-when-downgrade",
  "same-origin",
  "origin",
  "strict-origin",
  "origin-when-cross-origin",
  "strict-origin-when-cross-origin",
  "unsafe-url"
};
```

**Comment by Crevil** [comment]
@nigrosimone I'll look into making a PR on this if that's ok with you?

**Comment by szepeviktor** [comment]
Thanks.

**Comment by nigrosimone** [comment]
@Crevil yes thanks! it's very appreciated! The Security Headers feature is made by @amiga-500, you can review her work on this prs:

["Security Headers" Section Added to Browser Cache](https://github.com/szepeviktor/w3-total-cache-fixed/pull/344)<br>:wrench: [+ **Extra: #PR363** &ndash; Adds Default Values to CSP (Security Headers)](https://github.com/szepeviktor/w3-total-cache-fixed/pull/363)<br>:wrench: [+ **Extra: #PR377** &ndash; Important Change &ndash; Session Cookies](https://github.com/szepeviktor/w3-total-cache-fixed/pull/377)
